src/app/examples/dropdown/dropdown-with-overlay.html

```diff
@@ -25,8 +25,14 @@
     [cdkTrapFocus]="true"
     [cdkTrapFocusAutoCapture]="true"
   >
-    <button class="dropdown-item" type="button" (click)="logEvent('Action')">Action</button>
-    <button class="dropdown-item" type="button" (click)="logEvent('Action 2')">Action 2</button>
-    <button class="dropdown-item" type="button" (click)="logEvent('Action 3')">Action 3</button>
+    <button class="dropdown-item focus-inside" type="button" (click)="logEvent('Action')"
+      >Action</button
+    >
+    <button class="dropdown-item focus-inside" type="button" (click)="logEvent('Action 2')"
+      >Action 2</button
+    >
+    <button class="dropdown-item focus-inside" type="button" (click)="logEvent('Action 3')"
+      >Action 3</button
+    >
```

Comment:
@dauriamarco What's this change about? Related or unrelated?

Reply:
Not directly related to the button groups but related to this comment: https://github.com/siemens/element/pull/588#pullrequestreview-3134732076

I noticed that also there we had the same issue with the focus so I thought to quickly fix it.